orca/versions.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
```

Comment:
I believe this second `terraform` block is redundant (since `required_version` is specified on line 5).

Reply:
Removed: b056ef3